back/app/policies/cosponsors_initiative_policy.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class CosponsorsInitiativePolicy < ApplicationPolicy
+  class Scope
```

Comment:
Is `Scope` ever used?

Reply:
No, I just couldn't find any example in our codebase where we don't define it, and I figured it doesn't hurt to include it.

Reply:
Maybe yes. I asked because it's not exactly correct. It should not be `scope.all` for all users and it's not worth changing it if it's not used.

Reply:
Removed in #5578